### PR TITLE
Comment func

### PR DESCRIPTION
**Pull request by Developers-codz**

### CODE REVIEW

File: src/Redux/Reducers/postsSlice.js [hunk 1]
```diff
@@ -82,6 +82,26 @@ export const editPost = createAsyncThunk(
   }
 );
 
+export const commentPost = createAsyncThunk(
+  "posts/commentPost",
+  async ({id,comment}, { rejectWithValue }) => {
+    const encodedToken = localStorage.getItem("token");
+    // const {_id} = commentData;
```
Comment (ApurvaSawant11) [inline src/Redux/Reducers/postsSlice.js]: You can remove this

File: src/Redux/Reducers/postsSlice.js [hunk 2]
```diff
@@ -82,6 +82,26 @@ export const editPost = createAsyncThunk(
...
+    try {
+      const response = await axios.post(
+        `/api/posts/${id}/comment`,
+        { commentData: { content: comment } },
+        { headers: { authorization: encodedToken } }
+      );
+      console.log(response.data);
```
Comment (ApurvaSawant11) [inline src/Redux/Reducers/postsSlice.js]: There are multiple console.log and comments in this file. Please remove them

**Review by vishalg8454 — APPROVED** [review]
All looks good to merge, Clean UI.

File: src/pages/Explore/Explore.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  return (
+    <>
+      <div className="section">
+        <div>
+          {posts.map((post) => {
+            return <Post post={post} key={post._id} />;
+          })}
+        </div>
+      </div>
+    </>
```
Comment (ApurvaSawant11) [inline src/pages/Explore/Explore.jsx]: The empty fragment is not needed here. Also check if you really need two `div` tags or not

**Comment by Dev-AkshatGupta** [comment]
things seems cool to me you can merge it

File: src/components/posts/Post.jsx [hunk 4]
```diff
@@ -31,25 +33,45 @@ import {
...
-}
+  const userToFollow = users.find((user) => user.username === post.username);
+  const clickHandler = () => {
+    setOpen((open) => !open);
+    isMenuOpen ? dispatch(setEditEmpty()) : dispatch(getPostToEdit(post));
+  };
+  const posts = useSelector(store => store.posts.posts);
+ useEffect(()=>{
+  setCurrentPost(posts.find(currpost => currpost._id === post._id ))
+  console.log(currentPost)
```
Comment (ApurvaSawant11) [inline src/components/posts/Post.jsx]: console.log

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me! Just gave one suggestion.

**Comment by Developers-codz** [comment]
> Looks good to me! Just gave one suggestion.

thank you for detailed review, removed all the consoles and commented code

**Pull request merged**